2/debian-10/rootfs/opt/bitnami/scripts/libopenldap.sh

```diff
@@ -308,7 +308,7 @@ EOF
 ldap_add_custom_ldifs() {
     info "Loading custom LDIF files..."
     warn "Ignoring LDAP_USERS, LDAP_PASSWORDS, LDAP_USER_DC and LDAP_GROUP environment variables..."
-    debug_execute find "$LDAP_CUSTOM_LDIF_DIR" -maxdepth 1 -type f -iname '*.ldif' -print0 | sort -z | xargs --null -I{} ldapadd -f {} -H 'ldapi:///' -D "$LDAP_ADMIN_DN" -w "$LDAP_ADMIN_PASSWORD"
+    debug_execute find "$LDAP_CUSTOM_LDIF_DIR" -maxdepth 1 -type f,l -iname '*.ldif' -print0 | sort -z | xargs --null -I{} ldapadd -f {} -H 'ldapi:///' -D "$LDAP_ADMIN_DN" -w "$LDAP_ADMIN_PASSWORD"
```

Comment:
As I mentioned at https://github.com/bitnami/bitnami-docker-openldap/pull/8
It doesn't make much sense to pass the whole "find command" to debug_execute. Instead, we only need to run the "ldapadd command" with `debug_execute`.
This is a possible solution:
```suggestion
find "$LDAP_CUSTOM_LDIF_DIR" -maxdepth 1 -type f,l -iname '*.ldif' -print0 | sort -z | xargs --null -I{} bash -c ". /opt/bitnami/scripts/libos.sh && debug_execute ldapadd -f {} -H 'ldapi:///' -D $LDAP_ADMIN_DN -w $LDAP_ADMIN_PASSWORD"
```

Reply:
Fair enough. That's done in my latest commit.